region/crateria/west.json

```diff
@@ -801,7 +827,13 @@
               },
               {
                 "length": 18,
-                "requires": [ "canDestroyBombWalls" ],
+                "requires": [ 
+                  {"or": [
+                    "ScrewAttack",
```

Comment:
I must have missed this when doing obstacles. That leftmost bomb wall needs to be an obstacle, because being able to open up the runway by having PBs and not using any doesn't work.

Reply:
Ok guess I get to be Picasso again.